packages/flame/lib/src/collisions/hitboxes/shape_hitbox.dart

```diff
@@ -1,5 +1,6 @@
 import 'package:flame/collisions.dart';
 import 'package:flame/components.dart';
+import 'package:flame/extensions.dart';
```

Comment:
Why import extensions here? 😅

Reply:
It's instead of importing `dart:ui`, shouldn't really matter. :)

Reply:
Def not, was just curious haha